drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -297,4 +297,27 @@ config NRF700X_RSSI_STALE_TIMEOUT_MS
 	  value as driver does not store it and requires RPU to provide the
 	  info.
 
+if NETWORKING
+
+# Finetune defaults for certain system components used by the driver
+config HEAP_MEM_POOL_SIZE
+	default 120000
```

Comment:
Is this based on experiments to see lowest possible where Wi-Fi works? This might not work, `153000` is the number that we come up with , of course, we have a plan to run more tests and come up with a better number.

Reply:
It's just an arbitrary value that worked for me, I'm open to setting something more appropriate as well.